### ISSUE TITLE
The Basic Regression example breaks down in the Normalization section

### ISSUE THREAD

**Issue by sreedta8** [t-kalinowski/tensorflow.rstudio.com#65]
Using Reticulate, RStudio and Python (in a conda env) to run the basic regression example here [https://tensorflow.rstudio.com/tutorials/keras/regression](url)

When the code gets to the Normalization section, it breaks down. Specifically:

```normalizer <- layer_normalization(axis = -1L)``` does not work. Instead the code that worked for me was

```normalizer = tf.keras.layers.LayerNormalization(axis = -1)``` - when users are beginners such as myself, even though this ran, there is no way of knowing if it ran correctly.

The next place where this breaks down is

```normalizer %>% adapt(as.matrix(train_features))``` When I run this in a R Markdown file, I get an error message identifying the pipe symbol "%>%" as invalid syntax

I have not found a correction to this error. If anyone can help me correct this error or identify what I may be doing wrong it would be very helpful. I have been copying the commands verbatim, so there is no scope for error due to entering the wrong commands. If these errors because the commands have not been updated to reflect the changes to the libraries, I understand that too, since it is not easy to keep pace. I would like to help if possible

Thank you!
